iwyu_globals.cc

```diff
@@ -91,6 +91,8 @@ static void PrintHelp(const char* extra_msg) {
          "        the maximum line length can still be exceeded with long\n"
          "        file names (default: 80).\n"
          "   --no_comments: do not add 'why' comments.\n"
+         "   --force_comments: always add 'why' comments, even if no #include\n"
```

Comment:
Would it be worse to call this `--update_comments` too? Feels nice if there's some connection between the `fix_includes.py` args and the `include-what-you-use` args.